qa/tests/features/new/manual/dashboard/dashboard.feature

```diff
@@ -0,0 +1,125 @@
+@regression
+Feature: Filter refiner, Decision tree, Dashboard
+As an anfisa user I want to create custom layout for attributes
+
+Background: 
+	
+	Given Filter refiner or Decision tree of "xl_PGP3140_wgs_NIST-4_2" was opened
+
+@smoke
+Scenario: 01 Open dashboard
+	
+	When User clicks "Card view" button near the attribute search panel
+	Then Dashboard should be opened
+	And All attributes should be presented
+
+@smoke
+Scenario: 02 Close section tab
+
+	When User clicks "Card View" button near the attribute search panel
+	And Clicks opened section name or along the name 
+	Then Section should be closed
+	And Section should be moved at the bottom of the site with other sections
+
+@smoke
+Scenario: 03 Open section tab
+
+	When  User clicks closed section name or along the name
```

Comment:
In scenario 02 you fixed it, but in 03 and below user still doesn't switch to dashboard view before trying to click something on the panel